package.json

```diff
@@ -1,167 +1,167 @@
 {
-  "name": "Nebra",
-  "version": "1.0.4",
-  "build": "1",
-  "private": true,
-  "scripts": {
-    "android": "react-native run-android",
-    "ios": "react-native run-ios",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx && tsc",
-    "prettier": "prettier --write *.ts*",
-    "start": "react-native start",
-    "test": "jest",
-    "icon": "yarn icon-label && yarn icon-gen && yarn icon-cleanup",
-    "icon-label": "app-icon label -i icon.png -o output.png --bottom 3.0.0",
-    "icon-gen": "app-icon generate -i output.png",
-    "icon-cleanup": "rm output.png",
-    "android-clean": "cd android && ./gradlew clean && cd ..",
-    "ios-clean": "rm -rf ios/Pods && rm -rf ios/build",
-    "pod-install": "cd ios && pod install && cd ..",
-    "clean-install": "watchman watch-del-all && rm -rf $TMPDIR/react-native-packager-cache-* && rm -rf $TMPDIR/metro-bundler-cache-* && rm -rf node_modules/ && yarn cache clean && yarn install && yarn android-clean && yarn ios-clean && yarn pod-install",
-    "clean-start": "yarn start -- --reset-cache",
-    "postinstall": "yarn patch-package",
-    "bump-patch": "npx react-native bump-version --skip-semver-for android --skip-code-for all --type patch",
-    "bump-minor": "npx react-native bump-version --skip-semver-for android --skip-code-for all --type minor",
-    "bump-major": "npx react-native bump-version --skip-semver-for android --skip-code-for all --type major"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
+    "name": "Nebra",
```

Comment:
It's distracting when the indentation changes. Hard to make sure only the expected changes are included.

This is a JS package, so I'm inclined to say 2 spaces is more correct.